lb_content_resolver/database.py

```diff
@@ -56,7 +56,8 @@ def create(self):
             Create the database. Can be run again to create tables that have been recently added to the code,
             but don't exist in the DB yet.
         """
-
+        db_dir = os.path.dirname(os.path.realpath(self.db_file))
```

Comment:
This could still throw an exception (persimissions). If we're trying to get rid of stacktraces, we should catch these errors and show a nice error to the user.

Reply:
Done in [da6c0ae](https://github.com/metabrainz/listenbrainz-content-resolver/pull/18/commits/da6c0ae616e92b32ebb58ffdd2289bea02372196)